src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/BasicFields.drl

```diff
@@ -38,7 +38,7 @@ rule "Copy basic fields and agenda controller"
         workloadInventoryReport.setProduct(vmWorkloadInventoryModel.getProduct());
         workloadInventoryReport.setVersion(vmWorkloadInventoryModel.getVersion());
         workloadInventoryReport.setHost_name(vmWorkloadInventoryModel.getHost_name());
-        workloadInventoryReport.setCreationDate(new Date());
+        workloadInventoryReport.setCreationDate(vmWorkloadInventoryModel.getScanRunDate());
```

Comment:
Please add above (where i can not comment) in the `when` part, the condition to check the `scanRunDate` field is not null:
```
host_name != null,
scanRunDate != null
```

Reply:
+1